libs/angular/store/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@studiohyperdrive/ngx-store",
-	"version": "19.0.0",
+	"version": "20.0.0",
```

Comment:
This version should be `19.1.0`, because the major version will always follow the current major Angular version.